### DISCUSSION TITLE
Garbage collection and cache eviction

### DISCUSSION THREAD

**Discussion by kaanguru** [Akryum/vue-apollo#1035] [Q&A]
**How to clean all cache?**

Is it possible to use ` cache.gc();` ? 

**Expected behavior**
Clear all cache data

**Versions**
vue: 2.6.11
vue-apollo: 3.0.4
"apollo-boost": "^0.4.9",
"apollo-link-context": "^1.0.20",

**Comment by dmitry** [comment]
Try: `this.$apollo.defaultClient.cache.gc()`
